templates/zone.header.erb

```diff
@@ -11,3 +11,5 @@ $TTL <%= @ttl %>
 <% if ! @reverse %>
 <%= @soa %>. IN A <%= @soaip %>
 <% end %>
+
+$INCLUDE <%= @zonefilepath -%>/<%= @filename -%>.static <%= @zone -%>.
```

Comment:
I'd prefer if this was hidden under a boolean so most users wouldn't have this enabled.